### PR TITLE
Clarify Scale down rule in the documentation (FAQ.md)

### PR DESCRIPTION
**Pull request by faan11**
Specify the maximum value in the CA scale-down rule

#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
To clarify which criteria is currently used to scale down a node.

### CODE REVIEW

**Comment by Shubham82** [comment]
you have to sign the CLA before the PR can be reviewed.
See the following document to sign the CLA: [Signing Contributor License Agreements(CLA)](https://github.com/kubernetes/autoscaler/blob/master/CONTRIBUTING.md#signing-contributor-license-agreementscla)

**Comment by Shubham82** [comment]
To check EasyCLA

/easycla

File: cluster-autoscaler/FAQ.md
Comment (vadasambar) [inline cluster-autoscaler/FAQ.md:593]: suggestion: makes the wording less ambiguous 
```suggestion
* The maximum value between the sum of cpu requests and sum of memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
```

@faan11 can you also update [the help text for `--scale-down-utilization-threshold`](https://github.com/kubernetes/autoscaler/blob/c068feb5f0e1935bff241f4f2e7cb1f12c763281/cluster-autoscaler/main.go#L123). Would be useful to clarify this in the help text as well.

Comment (vadasambar) [reply cluster-autoscaler/FAQ.md:593]: Notes to prove what Fabio said is true : 

https://github.com/kubernetes/autoscaler/blob/57f2814267399e0900a70d567a7813089a3cfa5b/cluster-autoscaler/core/scaledown/eligibility/eligibility.go#L142-L156

https://github.com/kubernetes/autoscaler/blob/b569db410fd8bb152c4cc1a940f059c7e2bf0538/cluster-autoscaler/simulator/utilization/info.go#L61-L78

**Comment by vadasambar** [comment]
@faan11 thank you for the PR! Mostly LGTM except the nits and can you sign the CLA please :pray:

**Comment by faan11** [comment]
Hi [vadasamba](https://github.com/vadasambar), sorry for the delay,  i've accepted the Easycla license. 
Thank you for your suggested changes. I suppose, we only need to merge this pull request.

**Comment by vadasambar** [comment]
@faan11 can you address the review comments? Thanks.

**Comment by faan11** [comment]
Yes, I've committed them. Now it should be clearer.

**Comment by Shubham82** [comment]
@faan11, thanks for the changes, but you didn't update [the help text for --scale-down-utilization-threshold](https://github.com/kubernetes/autoscaler/blob/c068feb5f0e1935bff241f4f2e7cb1f12c763281/cluster-autoscaler/main.go#L123) as suggested by @vadasambar.

**Comment by faan11** [comment]
Thank you @Shubham82, I apologize for misreading.  I only read the first change. As suggested by @vadasambar , i provided  a proper description for the flag scale-down-utilization-threshold . Let me know if it fine or we need to rephrase it somehow.

**Review by vadasambar — CHANGES_REQUESTED** [review]

**Comment by vadasambar** [comment]
@faan11 almost there. Let me know when you are done with them.

**Comment by faan11** [comment]
Hi @vadasambar, thank you for your tips. I updated the file as you suggested, In the FAQ.md, i have also specified that the value is a floating point value that can range between 0 and 1.

Let me know if it is ok or you have additional pointers to improve this pull request.
Thank you

**Comment by Shubham82** [comment]
@faan11, squash your  commits.

**Comment by vadasambar** [comment]
@faan11 can you squash your commits? Thank you! Otherwise LGTM

/lgtm
/hold

**Comment by faan11** [comment]
Yes i'll squash the branch commits and notify you when i'm done with it.

**Comment by faan11** [comment]
I suppose we can proceed now @vadasambar  @Shubham82

**Review by vadasambar — APPROVED** [review]

**Comment by vadasambar** [comment]
Thank you for the PR @faan11 ! 

@BigDarkClown @x13n can we get approval on this :pray:

**Comment by faan11** [comment]
HI @BigDarkClown @x13n,

Can you please review this issue?

Thank you

**Comment by vadasambar** [comment]
> HI @BigDarkClown @x13n,
> 
> Can you please review this issue?
> 
> Thank you

I will bring this PR up in the upcoming sig meeting on Monday (8th Jan) if you don't get an approval by then.

File: cluster-autoscaler/FAQ.md
Comment (x13n) [inline cluster-autoscaler/FAQ.md:593]: Rather than talking about max of sums, wouldn't it be clearer to say that *both* values have to be below the threshold?

**Review by x13n — COMMENTED** [review]
Just one minor comment, feel free to cancel the hold if you disagree.

/lgtm
/approve
/hold

File: cluster-autoscaler/FAQ.md
Comment (faan11) [reply cluster-autoscaler/FAQ.md:593]: hi @x13n , it sounds good, i'll merge your contribution as well. Thank you  :)

Comment (faan11) [reply cluster-autoscaler/FAQ.md:593]: hi @x13n , it sounds good, i'll merge your contribution as well. Thank you  :)

Comment (faan11) [reply cluster-autoscaler/FAQ.md]: Hi @x13n, the change has been applied only in this section. The flag definitions were left as is for the sake of clarity.

Comment (Shubham82) [reply cluster-autoscaler/FAQ.md]: Thanks @faan11
It looks good to me.
/lgtm

Comment (Shubham82) [reply cluster-autoscaler/FAQ.md]: removing the `/hold` label as the above suggestion is addressed.

Comment (Shubham82) [reply cluster-autoscaler/FAQ.md]: /unhold

**Pull request merged**
